include/faabric/util/bytes.h

```diff
@@ -25,6 +27,20 @@ int safeCopyToBuffer(const uint8_t* dataIn,
                      uint8_t* buffer,
                      int bufferLen);
 
+/*
+ * Returns a list of pairs of <start, length> for any bytes differing between
+ * the two arrays.
+ */
+std::vector<std::pair<uint32_t, uint32_t>> diffArrayRegions(
+  std::span<const uint8_t> a,
+  std::span<const uint8_t> b);
+
+/*
+ * Returns a list of flags marking which bytes differ between the two arrays.
+ */
+std::vector<bool> diffArrays(std::span<const uint8_t> a,
```

Comment:
Isn't this a bit-wise XOR?

Reply:
It's currently byte-wise, not bit-wise, but this may change. I noticed that this function isn't actually used, so I should probably just delete it.